Update the examples in the Search API v1 docs (#78)
opendatasoft/ods-documentation-explore-api: source/includes/v1/query_language_and_geo_filtering.md

# Query Language and Geo Filtering

Filtering features are built in the core of the Opendatasoft API engine. Many of the previously listed APIs can take as

> parameters filters for constraining the list of returned datasets or records.

Reviewer: Maybe worth it to re-organize the sentence or add commas as I had to read it 3 times to get the sence xD

Author: It's true that this sentence is not crystal clear. I added a better [suggestion]( https://github.com/opendatasoft/ods-documentation-explore-api/pull/78/files/f400478e6c7091431b86c26f9b67fc9c66b6edc6#r576330398).

> Filtering features are built in the core of the Opendatasoft API engine. Many of the previously listed APIs can take as
> parameters filters for constraining the list of returned datasets or records.

Author: Based on @NTerpo's comment, the following rewording makes more sense.
```suggestion
Filtering features are built in the core of the Opendatasoft API engine. Many of the previously listed APIs can take filters as parameters, so that the response only contains the datasets or records you want.
```
